### PR TITLE
[ios sdk] add interp-only support

### PR DESCRIPTION
**Pull request by lewurm**
depends on https://github.com/mono/mono/pull/9295

should be backported to `2018-06`

### CODE REVIEW

File: sdks/ios/README.md [hunk 1]
```diff
@@ -68,3 +68,7 @@ seem to read the device logs, only the graphical Console app can.
 * Use a publish-subscribe pattern by uploading test results to some cloud service like
 Azure EventHub. This only requires client side internet access on the device and
 the test harness.
+
+## Notes
+
+Bernhard needed this patch to make it work on his machine: https://gist.github.com/lewurm/7711187deadb57bae97a4d4be4df563b
```
Comment (vargaz) [inline sdks/ios/README.md]: This should probably not be part of the README.

Comment (vargaz) [reply sdks/ios/README.md]: Then it should cleaned up and included here, not referring to a gist somewhere.

Comment (lewurm) [reply sdks/ios/README.md]: We should probably generate `sdks/ios/runtime/Entitlements.xcent`, but let's do that in another PR.

**Comment by lewurm** [comment]
@monojenkins squash

**Pull request merged**